config.go

```diff
@@ -292,6 +293,11 @@ func loadConfig() (*config, error) {
 		return nil, errors.New("invalid vspfee - should be greater than 0.01 and less than 100.0")
 	}
 
+	// If VSP is not closed, ignore any provided closure message.
+	if !cfg.VspClosed {
+		cfg.VspClosedMsg = ""
```

Comment:
Not sure why this needs to be set to nil if there is a flag available which can be checked before the message is used.

Reply:
This same `if` statement will still need to exist in `vspinfo.go` to ensure that the message is only included in `/vspinfo` response when the VSP is actually closed.

I put it in `config.go` instead because the value of `VspClosedMsg` should never be used when `VspClosed == true`, so putting it here will prevent the need for it to be duplicated in other places which might access the closed message in future.

Reply:
I get that, but ignoring the closure message by setting it to an empty string isn't intuitive to me, if the checking the flag precedes using the closure message then it'd only be displayed if the check passes anyway so why mutate the config value? Plus the check isn't expensive. The current approach works and I do not have a strong opinion on this, take another look however.